paywall/src/utils/routes.js

```diff
@@ -16,17 +17,26 @@ export const lockRoute = path => {
     }
   }
 
+  let account = url.hash && url.hash.substring(1)
+  const matchAccount = account.match(ACCOUNT_REGEXP)
+  account = matchAccount ? matchAccount[0] : undefined
```

Comment:
`account = matchAccount ? matchAccount[0] : undefined` maybe we can just use `account = matchAccount[0]` (I know `null` is not `undefined` but then you need to explain why you need `undefined` because future-us will not know what you're doing here.

Reply:
this is actually an attempt to unify the output from `lockRoute`, but I think I will make the lines below return `null` instead if falsy instead of the current `undefined`. That will require changing the `baseRoute` in tests, but I think it is more consistent and less weird. Because I agree, explicitly using `undefined` smells bad :).